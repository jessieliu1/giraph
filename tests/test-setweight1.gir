int main() {
	wegraph<int> g = [A:1 -{5}- B -{6}- C];
	for_edge (e : g) {
		e.set_weight(e.weight() + 1);
	}
	for_edge (e : g) {
		print(e.weight());
	}
	print(g.get_edge_weight(C, B));
	print(g.get_edge_weight(B, C));

	wedigraph<int> g2 = [A -{5}-> B <-{6}- C <-{8}- B];
	for_edge (e : g2) {
		e.set_weight(e.weight() + 1);
	}
	for_edge (e : g2) {
		print(e.weight());
	}
	print(g2.get_edge_weight(C, B));
	print(g2.get_edge_weight(B, C));

	return 0;
}