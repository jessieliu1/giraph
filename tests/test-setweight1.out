6
7
7
7
6
7
9
7
9
